### PR TITLE
i18n(de): update fonts.mdx

### PR DESCRIPTION
**Pull request by DerTimonius**
#### Description (required)

This PR updates the german translation of `fonts.mdx` as it is shown as outdated on the translation tracker

### CODE REVIEW

File: src/content/docs/de/guides/fonts.mdx [hunk 1]
```diff
@@ -106,16 +101,40 @@ Das [Fontsource](https://fontsource.org/)-Projekt ermöglicht die Verwendung von
...
+## Schriftarten mit Tailwind hinzufügen
+
+Falls du die [Tailwind-Integration](/de/guides/integrations-guide/tailwind/) verwendest, kannst du entweder eine [`@font-face`-Regel für eine lokale Schriftart](#verwenden-einer-lokalen-schriftdatei) hinzufügen oder die [`Import`-Strategie von Fontsource](#verwenden-von-fontsource) verwenden, um die Schriftart zu registrieren.
 
-### Schriftarten mit Tailwind hinzufügen
+Um deine Schriftart in Tailwind zu registrieren:
 
-Falls du die [Tailwind-Integration](/de/guides/integrations-guide/tailwind/) verwendest, kannst du entweder eine `@font-face`-Regel für eine lokale Schriftart hinzufügen oder die `Import`-Strategie von Fontsource verwenden, um die Schriftart zu registrieren. Folge dann [Tailwinds Dokumentation zum Hinzufügen benutzerdefinierter Schriftfamilien](https://tailwindcss.com/docs/font-family#using-custom-values).
+1. Folge einer der Anleitungen oben, lasse aber den Schritt, die `font-family` in deinem CSS-Code hinzuzufügen, aus.
+2. Füge den Schriftartnamen in der `tailwind.config.mjs` Datei hinzu
```
Comment (1p0d) [inline src/content/docs/de/guides/fonts.mdx:111]:
```suggestion
2. Füge den Schriftartnamen in der `tailwind.config.mjs` Datei hinzu.
```

**Review by 1p0d — COMMENTED** [review]
LGTM, would just add a full stop.

**Comment by yanthomasdev** [comment]
Thanks @DerTimonius, could I ask you to add this change in here as well? https://github.com/withastro/docs/commit/9ebd57629d3182be3b7ef2830e2621fc819d2d07 - That should be all needed to get this merged. 🙌

**Comment by DerTimonius** [comment]
@yanthomasdev Done!

**Review by yanthomasdev — APPROVED** [review]
LGTM! Thanks 🙌

**Pull request merged**